src/test/java/PriceManagerTest.java

```diff
@@ -56,61 +67,61 @@ public void addItemsToInventoryAndAssertAgainstThePrice() {
         Inventory milk = new Inventory("1", "Milk", 5.0, 1, false);
         Inventory corona = new Inventory("2", "Corona", 11.25, 1, false);
         Inventory whisky = new Inventory("3", "Ardbeg", 526.0, 1, false);
-        superMarket.addItem(milk);
-        superMarket.addItem(corona);
-        superMarket.addItem(whisky);
+        superMarket.addItemToInventory(milk);
+        superMarket.addItemToInventory(corona);
+        superMarket.addItemToInventory(whisky);
         assertThat(milk.itemPrice, is(5.0));
     }
 
     @Test
-    public void calculateTotalValueOfItemsAddedToTheInventory() {
-        Inventory milk = new Inventory("1", "Milk", 5.0, 1, false);
-        Inventory corona = new Inventory("2", "Corona", 11.25, 1, false);
-        Inventory whisky = new Inventory("3", "Ardbeg", 526.0, 1, false);
-        Inventory bread = new Inventory("4", "Bread", 12.0, 1, false);
-        superMarket.addItem(milk);
-        superMarket.addItem(corona);
-        superMarket.addItem(whisky);
-        superMarket.addItem(bread);
+    public void calculateTotalValueOfItemsAddedInTheCart() {
+        initializeInventory();
+        includePromotions();
+        Cart milk = new Cart("Milk", 1);
+        Cart corona = new Cart("Corona", 1);
+        Cart whisky = new Cart("Whisky", 1);
+        Cart bread = new Cart("Bread", 1);
+        superMarket.addItemsToTheCart(milk);
+        superMarket.addItemsToTheCart(corona);
+        superMarket.addItemsToTheCart(whisky);
+        superMarket.addItemsToTheCart(bread);
         assertThat(superMarket.totalValueOfCart(), is(554.25));
     }
 
     @Test
     public void totalNumberOfItemsInTheCart() {
-        Inventory milk = new Inventory("1", "Milk", 5.0, 1, false);
-        Inventory corona = new Inventory("2", "Corona", 11.25, 1, false);
-        Inventory whisky = new Inventory("3", "Ardbeg", 526.0, 1, false);
-        Inventory bread = new Inventory("4", "Bread", 12.0, 1, false);
-        superMarket.addItem(milk);
-        superMarket.addItem(corona);
-        superMarket.addItem(whisky);
-        superMarket.addItem(bread);
+        initializeInventory();
+        Cart milk = new Cart("Milk", 1);
+        Cart corona = new Cart("Corona", 1);
+        Cart whisky = new Cart("Whisky", 1);
+        Cart bread = new Cart("Bread", 1);
+        superMarket.addItemsToTheCart(milk);
+        superMarket.addItemsToTheCart(corona);
+        superMarket.addItemsToTheCart(whisky);
+        superMarket.addItemsToTheCart(bread);
         assertThat(superMarket.itemsInTheCart(), is(4));
     }
 
     @Test
     public void verifyPromotionsOnTheItemsAddedToTheCart() {
```

Comment:
If you deleted `calculatePromotion()` method body, delete also a test (or at least comment it out) because it will be not possible to merge PR.

Reply:
Removed as we need a better implementation of managing promotions than the one previously available